gettext-sys/lib.rs

```diff
@@ -26,7 +26,7 @@ extern "C" {
 
     pub fn bindtextdomain(domain: *const c_char, dir: *const c_char) -> *mut c_char;
     #[cfg(windows)]
-    pub fn wbindtextdomain(domain: *const c_char, dir: *const wchar_t) -> *mut wchar_t;
+    pub fn libintl_wbindtextdomain(domain: *const c_char, dir: *const wchar_t) -> *mut wchar_t;
```

Comment:
It'd be nice to keep this crate in line with [GNU gettext's docs](https://www.gnu.org/software/gettext/manual/gettext.html), which don't mention `libintl_wbindtextdomain`. Let's emulate the approach that the upstream took: make this definition private, and provide a `pub fn wbindtextdomain` that calls the private version. Then drop the changes from gettext-rs, and everything would be golden :)

Reply:
Done